torch/onnx/symbolic_opset11.py

```diff
@@ -390,6 +390,12 @@ def sort(g, self, dim, decending, out=None):
     return sym_help._sort_helper(g, self, dim, decending=decending, out=out)
 
 
+@parse_args("v", "i", "i")
+def argsort(g, self, dim, decending):
```

Comment:
Please add relevant test cases. Example test cases found in test/onnx/test_pytorch_onnx_onnxruntime.py

Reply:
Added **`test_argsort`** and **`test_argsort_ascending`**. They use the same syntax as the existing **`test_sort`** and **`test_sort_ascending`**.